### PR TITLE
[wicketd] Install real host phase 1 after installinator completes

### PR DESCRIPTION
**Pull request by jgallagher**
Actually waiting for installinator to complete is still a placeholder as of this PR, but this fleshes out what to do after it's in place.

### CODE REVIEW

File: wicketd/src/update_tracker.rs [hunk 1]
```diff
@@ -409,8 +409,14 @@ impl UpdateDriver {
         // TODO wait for installinator completion, ideally with progress along
         // the way.
 
-        // TODO power down host and deliver matching host phase 1
-        _ = plan.host_phase_1;
+        // Installinator is done: install the host phase 1 that matches the host
+        // phase 1 it installed, and boot our newly-recovered sled.
```
Comment (andrewjstone) [inline wicketd/src/update_tracker.rs:413]: typo: `host phase 2 it installed` ?

File: wicketd/src/update_tracker.rs [hunk 2]
```diff
@@ -598,14 +604,75 @@ impl UpdateDriver {
...
+        // can set the correct boot device. Thinking out loud: Even if it
+        // doesn't do it today, installinator probably wants to _dynamically_
+        // choose an M.2 to account for missing or failed drives. Maybe its
+        // final completion message should tell us which slot (or both!) it
+        // wrote to, and then we echo that choice here?
+        let host_phase_1_boot_slot = 0;
+        self.deliver_host_phase1(&plan.host_phase_1, host_phase_1_boot_slot)
+            .await?;
+
+        // Clear the installinator image ID; failing to do this is _not_ fatal,
```
Comment (andrewjstone) [inline wicketd/src/update_tracker.rs:635]: Another great, and very useful comment for future archeologists!

**Review by andrewjstone — APPROVED** [review]
🚀

File: wicketd/src/update_tracker.rs (hunk 1 above)
Comment (jgallagher) [reply wicketd/src/update_tracker.rs:413]: Yep, thanks!

**Pull request merged**